### PR TITLE
Add tip link to Polkassembly

### PR DESCRIPTION
**Pull request by Tbaut**
Closes https://github.com/polkadot-js/apps/issues/3157
Looks like this:
image

### CODE REVIEW

**Comment by jacogr** [comment]
Exactly what I was actually complaining about yesterday (in private chats), that take is way too busy and now starts breaking lines as well. Not your issue, but basically -

- those links are way too obtuse and basically almost unusable
- all these tables are too busy
- ... not sure where to put them though

So complaining, been driving me crazy for a while, no solutions...

File: packages/page-treasury/src/Overview/Tip.tsx [hunk 1]
```diff
@@ -157,6 +157,13 @@ function Tip ({ bestNumber, className = '', hash, isMember, members, tip }: Prop
           />
         )}
       </td>
+      <td className='mini ui--media-1000'>
```
Comment (jacogr) [inline packages/page-treasury/src/Overview/Tip.tsx:160]: We need this is the header as well. (Should also adjust the classes, maybe 1200 or 1400 until we can sort out the busy nature)

Comment (Tbaut) [reply packages/page-treasury/src/Overview/Tip.tsx:160]: ah right totally missed it.

Comment (jacogr) [reply packages/page-treasury/src/Overview/Tip.tsx:160]: Don't worry, I miss that a lot since they stuff is in 2 places - so don't feel all alone.

**Comment by Tbaut** [comment]
> not sure where to put them though

A couple idea from the top of my head:
I could see these live in a "..." menu, it's 1 more click but would allow to show the full name of the links. Not sure how much they are used.

In terms of user friendliness, the platform's logo could also help keep these short, yet give an idea right away where they go.

**Comment by jacogr** [comment]
Agreed, the logos are actually (not a solution) but would make is much cleaner and much less obtuse. I actually use these a lot, not sure about others...

But the table issue is not confined to here, referendums are equally bad. As is accounts. And that is what makes it tricky since it needs to be a consistent solution. Just a lot of info to convey.

**Comment by Tbaut** [comment]
Just realized now what those `media` classes are. From what I could see, the table doesn't shrink past 150x, so the links wouldn't even be visible, I went for 1600 then.

**Comment by jacogr** [comment]
The media here is rough - it helps, but table are... problematic with the large amount of data deemed „useful“. For instance on tips the deposit field is actually useless with no real value. Can be removed.

**Pull request merged**